frontend/packages/ceph-storage-plugin/src/components/modals/block-pool-modal/modal-footer.tsx

```diff
@@ -160,7 +160,7 @@ export const BlockPoolModalFooter = (props: BlockPoolModalFooterProps) => {
             variant={buttonProp.variant}
             isDisabled={buttonProp.disable}
             id={buttonProp.id}
-            data-test={buttonProp.id}
+            data-test-id={buttonProp.id}
```

Comment:
`data-test` is the preferred method. Why use `id`?

Reply:
integration-tests-cypress/views/modal.ts is still using byLegacyTestID (cy.byLegacyTestID('modal-cancel-action').should('not.exist')).

same pool creation form is used between block pool create a page under OCS operator and block pool create modal under SC. So I am creating a footer manually, just to use this cypress views/modal.ts functions i am keeping it as legacy id